app/layout/statics/column/component.tsx

```diff
@@ -34,7 +36,22 @@ export const StaticColumn: React.FC<StaticColumnProps> = ({
           {description}
         </p>
       )}
-      <ButtonLink caption={caption} href={href} external={external} />
+      {image && (
+        <div>
+          <img
+            src={image}
+            alt=""
+            style={{
+              width: '403px',
```

Comment:
I'm afraid of this magic numbers

Reply:
Me too. I usually don't like it but I also don't see a good way to do it, a good way to add these images with the size the designers want. I could probably put some percentages, but I'm not sure this is a whole lot better... WDYT @mbarrenechea ?